src/fe-gtk/setup.c

```diff
@@ -2132,6 +2132,7 @@ setup_apply_real (int new_pix, int do_ulist, int do_layout, int do_identd)
 
 	mg_apply_setup ();
 	tray_apply_setup ();
+	hexchat_reinit_timers ();
```

Comment:
Note that `hex_gui_lagometer` can be toggled from the menu and that `hex_net_ping_timeout` is not exposed anywhere so that will no longer work.